src/Symfony/Component/DependencyInjection/Loader/PhpFileLoader.php

```diff
@@ -36,6 +36,7 @@
 class PhpFileLoader extends FileLoader
 {
     protected bool $autoRegisterAliasesForSinglyImplementedInterfaces = false;
+    private ?\Closure $configBuilderAutoloader = null;
```

Comment:
Could you confirm this property is necessary for recursive calls to the loader? As [pointed by Stof](https://github.com/symfony/symfony/pull/61940#discussion_r2402369851).